addons/slack/template/index.js

```diff
@@ -3,6 +3,14 @@ const health = require('./health.js');
 const Sdk = require('@fusebit/add-on-sdk');
 const { WebClient } = require('@slack/web-api');
 
+async function getStorageClient(ctx) {
+  return await Sdk.createStorageClient(
```

Comment:
Unnecessary await.

Reply:
We discussed with Tomek, he wanted to keep the `async` here for symmetry and future-proofing. If I don't `await` this it ends up `undefined` I think, so won't fix.

Reply:
Fixed